jest/defs.bzl

```diff
@@ -47,12 +52,13 @@ def jest_test(
         name,
         config = None,
         data = [],
-        snapshots = [],
+        snapshots = None,
         run_in_band = True,
         colors = True,
         auto_configure_reporters = True,
         auto_configure_test_sequencer = True,
-        snapshot_ext = ".snap",
+        snapshots_ext = ".snap",
+        silent_on_success = False,
```

Comment:
why default false?

Reply:
I think this is the expected behavior of web developers and what they are use when they run `jest -u` as they see the output of jest in that case.
 
This only applies to the jest snapshot generating target which is only built when you `bazel run //path/to:test_update_snapshots`. Perhaps a better name for this attribute such as `verbose_snapshot_updates`?

Reply:
Updated to
```
        quiet_snapshot_updates: When True, snapshot update stdout & stderr is hidden when the snapshot update is successful.

            On a snapshot update failure, its stdout & stderr will always be shown.
```
How's that?